docs/site/content/docs/upgrade-managed.md

```diff
@@ -0,0 +1,247 @@
+# Upgrade Management & Workload Cluster
+
+The management cluster upgrade process first upgrades the Cluster API providers for vSphere, Amazon EC2, or Azure that are running in the management cluster. Then, it upgrades the version of Kubernetes in all of the control planes and worker nodes of the management cluster.
+
+## Prerequisites
+
+- vSphere: If you are upgrading clusters that run on vSphere, the appropriate base image template OVAs must be available in vSphere as VM templates. For more information, see steps 1, and 2 in the [Prepare to Deploy a Management Cluster to vSphere](https://tanzucommunityedition.io/docs/v0.11/vsphere/#procedure).
+- Amazon EC2: If you are upgrading clusters that run on Amazon EC2, the Amazon Linux 2 Amazon Machine Images (AMI) that include the supported Kubernetes versions are publicly available to all Amazon EC2 users, in all supported AWS regions. Tanzu Community Edition automatically uses the appropriate AMI for the Kubernetes version that you specify during an upgrade.
+- Azure: If you are upgrading clusters that run on Azure, you must accept the terms for the new default VM image and each non-default VM image that you plan to use for your cluster VMs. To accept the terms, complete the following steps:
+
+  1. List all available VM images in the Azure Marketplace:
+
+        ```sh
+        az vm image list --publisher vmware-inc --offer tkg-capi --all
+        ```
+
+  1. Accept the terms for the new default VM image:
+
+        ```sh
+        az vm image terms accept --urn publisher:offer:sku:version
+        ```
+
+  1. For example, to accept the terms for k8s-1dot21dot2-ubuntu-2004, run:
+
+        ```sh
+        az vm image terms accept --urn vmware-inc:tkg-capi:k8s-1dot21dot2-ubuntu-2004:2021.05.17
+        ```
+
+## Upgrade Management Cluster
+
+1. Run the tanzu login command to see an interactive list of management clusters available for upgrade.
+
+    ```sh
+    tanzu login
+    ```
+
+    To change your current login context, use your up and down arrow keys to highlight a management cluster and then press Enter.
+
+1. The `tanzu login` command does not automatically set the kubectl context, run the following commands to set the context.
+
+    First, capture the management cluster’s kubeconfig and take note of the command for accessing the cluster in the output message:
+
+    ```sh
+    tanzu management-cluster kubeconfig get <MGMT-CLUSTER-NAME> --admin
+    ```
+
+    Set your kubectl context to the management cluster:
+
+    ```sh
+    kubectl config use-context <MGMT-CLUSTER-NAME>-admin@<MGMT-CLUSTER-NAME>
+    ```
+
+1. Run the `tanzu mc upgrade` command:
```

Comment:
I wonder if we should use the full `management-cluster` name instead of `mc` in our example commands.